templates/repository-template/README.md

```diff
@@ -1,23 +1,35 @@
-# Project Title
+> **NOTE:** It is a general template that can be used for a project README.md, example README.md, or any other README.md type in all Kyma repositories in the Kyma organization. Not all the sections are mandatory. Use only those that suit your use case but keep the proposed section order.
```

Comment:
if we say "not all of them are mandatory", we might want to point out specifically which ones are optional. Or are all of them optional? Then we need to rephrase the "not all are mandatory" sentence.

Reply:
I guess all of them are optional (maybe except for the Project Title and Overview?). I will mark that somehow :)

Reply:
Then I'd just add "(mandatory)" behind those two :D